fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/database/JdbcJavaType.java

```diff
@@ -30,14 +30,14 @@ public enum JdbcJavaType {
 
         @Override
         public Object toJdbcValueImpl(@NotNull DatabaseType dialect, Object value) {
-            return Boolean.TRUE.equals(value) ? 1 : 0;
+            return value == null ? null : (Boolean.TRUE.equals(value) ? 1 : 0);
```

Comment:
Previously the default was false. Are you sure it wise to change?

Reply:
there is no place for default on this level, previously it was not correct